microsoftgraph/microsoft-graph-explorer-v4
# Feature: Carry Login information from Try-It

## Overview

Carries the access token through the url from try-it to the full experience

Fixes #421 
Fixes #425 

### Notes

N/A

## Testing Instructions

* Load try it 
* Log in to the try-it experience
* Click the link to the full graph explorer (change the url to the local version to see it locally)
* Notice the access token in the url
* Y Notice that you are now signed in using the account you were signed in with previously
